src/main/scala/firrtl/stage/package.scala

```diff
@@ -33,6 +33,7 @@ package object stage {
           case InfoModeAnnotation(i)             => c.copy(infoModeName = i)
           case FirrtlCircuitAnnotation(cir)      => c.copy(firrtlCircuit = Some(cir))
           case a : CompilerAnnotation => logger.warn(s"Use of CompilerAnnotation is deprecated. Ignoring $a") ; c
+          case SuppressScalaVersionWarning       => c
```

Comment:
It's a-okay to do it this way, but it may make sense to add a `FirrtlOption` for this that can then be viewed as `FirrtlOptions`.

Reply:
I'm fine with adding it, I think I'll merge this PR as is but if you think we should add, can open a PR tomorrow